configurations/default.py

```diff
@@ -21,7 +21,7 @@
 
     # Cache engines are diskcache, redis, or memory if not specified
     "cache": {
-        "engine": "diskcache", 
+        "engine": "", 
```

Comment:
Why don't you like diskcache?

Reply:
It makes testing painful...this can be reinstated.

Reply:
If you want we can add a flag so that reductus server can be started with cache-engine == "memory"?

Reply:
You can also make a "testing" config JSON file that you load with --config-file

Reply:
I think that's a good idea. I find when I'm writing new modules I don't want bad results cached, so being able to turn this off easily would be a nice feature.

Reply:
Ok, added the `--cache-engine` flag to run.py, so you can do `reductus --cache-engine=memory` from the command line. c7605dc0d6a286b5caa2123eb4d43b7d993df4d9